src/ngraph/frontend/onnx_import/op/acos.hpp

```diff
@@ -32,7 +33,7 @@ namespace ngraph
             {
                 inline NodeVector acos(const Node& node)
                 {
-                    return {std::make_shared<ngraph::op::Acos>(node.get_ng_inputs().at(0))};
+                    return {std::make_shared<ngraph::opset1::Acos>(node.get_ng_inputs().at(0))};
```

Comment:
This is another approach (without `default_opset.hpp` file), include `ngraph/opsets/opset1.hpp` and use `ngraph::opset1` namespace.